src/interaction/VSpherePair.hpp

```diff
@@ -96,8 +97,7 @@ namespace espressopp {
       real getEpsilon() const { return epsilon; }
 
       real _computeEnergySqrRaw(real distSqr, real sigmaij) const {
-    	real rij = sqrt(distSqr);
-        real energy = ef1*pow(1.0L*sigmaij, 1.0L*mth)*exp(mth*rij/sigmaij);
+        real energy = ef1*pow(1.0L*sigmaij, 1.0L*mth)*exp(mth*distSqr/sigmaij);
```

Comment:
Yes. I considered \beta is just 1.
Then, I will put this information in the interaction/VSpherePair.py.